Tests/RxSwift/fromAsyncTests.swift

```diff
@@ -36,4 +44,47 @@ class FromAsyncTests: XCTestCase {
 
         XCTAssertEqual(observer.events, correct)
     }
+    
+    func testSingleResultEqualitySuccessCase() {
```

Comment:
basically another test case where you do `let result: String? = "result"` would be great. The rest of the test should probably remain the same